bootstrap.py

```diff
@@ -52,7 +52,7 @@
 
 # TODO: this mimics the override in the "common" job in .circleci/config.yaml
 with open('.circleci/common.sh', 'w') as fout:
-    fout.write("MINICONDA_VER=4.5.4\nBIOCONDA_UTILS_TAG=master\n")
+    fout.write("MINICONDA_VER=4.6.14\nBIOCONDA_UTILS_TAG=v0.16.11\n")
```

Comment:
You might want to revert this, since it isn't part of the recipe.

Reply:
Oops!  Thank you.